docs/getting_started/discover_ui.md

```diff
@@ -69,7 +88,57 @@ You can also use the **Content Browser** to search the Repository for content, e
 !!! note
 
     If you have administrator permissions, you can also view and manage User accounts and site skeletons in the **Admin** tab.
-    
+
     For more information about users and permissions, see [Users](../permission_management/permissions_and_users.md).
-    
+
     For more information about site skeletons, see [Site skeletons]([[= developer_doc =]]/multisite/site_factory/site_factory_configuration/#site-skeletons).
+
+## Focus mode
+
+Focus mode helps editors focus on information that is only relevant to their line of work and omit technical details that would otherwise introduce distraction.
+It is enabled when you first log into [[= product_name =]], and influences the way information is displayed in the user interface.
+If you need access to certain settings or technical details, you can [disable the focus mode](#disable-focus-mode) at any time.
```

Comment:
Once you use disable focus mode, once turn off. I think it should be consistent. Maybe change all entries with turn off to disable focus mode.